src/recipes/heartbeat-by-user-first-impression/config.js

```diff
@@ -213,7 +131,6 @@ module.exports = {
       locales:  supportedLocales
     },
     "aurora": {
-      "localeMultiplier": localeMultiplier, //TODO: Remove this hacky, temp code
```

Comment:
I am okay with this.  If we need to do more locale work, we can just get this back in.